### PR TITLE
Features/pages need body

### PR DESCRIPTION
**Pull request by knewter**
Add a body attribute to pages...

### CODE REVIEW

File: delivery/backend/app/conductors/pages/accepts_page_form.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+require 'interactors/pages/builds_page'
+require 'abc-adapters'
+require 'abc/page_presenter'
+
+module Abc
+  module Conductors
+    module Pages
+      class AcceptsPageForm < ::Afterburner::Framework::BaseConductor
+        def to_response
+          { page: presenters.pages.new(saved_page) }
```
Comment (robyurkowski) [inline delivery/backend/app/conductors/pages/accepts_page_form.rb:11]: What is this cursed one-point-ninery?!

Comment (robyurkowski) [inline delivery/backend/app/conductors/pages/accepts_page_form.rb:11]: Where is my hashrocket?!

**Comment by robyurkowski** [comment]
Other than the new ruby syntax (we've been pretty consistent with keeping the old), this is a cool PR. :+1:

**Comment by parndt** [comment]
FAIL https://codeclimate.com/github/grounded/afterburnercms

**Comment by knewter** [comment]
:(

**Comment by parndt** [comment]
:heart:

**Pull request merged**